wp-content/plugins/core/src/Assets/Theme/Scripts.php

```diff
@@ -119,7 +119,7 @@ public function enqueue_scripts(): void {
 			$this->localize_scripts( (string) reset( $handles ) );
 
 			if ( defined( 'HMR_DEV' ) && HMR_DEV === true ) {
-				wp_enqueue_script( 'tribe-scripts-hmr-bundle', 'http://localhost:3000/app.js', $handles, time(), true );
+				wp_enqueue_script( 'tribe-scripts-hmr-bundle', home_url() . ':9000/app.js', $handles, time(), true );
```

Comment:
where does `home_url` come from?

Reply:
WordPress Options table homeurl value. Or the environment variable `define('WP_HOME')`